pkg/golinters/sloglint/sloglint.go

```diff
@@ -14,14 +14,17 @@ func New(settings *config.SlogLintSettings) *goanalysis.Linter {
 		opts = &sloglint.Options{
 			NoMixedArgs:    settings.NoMixedArgs,
 			KVOnly:         settings.KVOnly,
-			NoGlobal:       settings.NoGlobal,
 			AttrOnly:       settings.AttrOnly,
-			ContextOnly:    settings.ContextOnly,
+			NoGlobal:       settings.NoGlobal,
+			ContextOnly:    settings.Context,
 			StaticMsg:      settings.StaticMsg,
 			NoRawKeys:      settings.NoRawKeys,
 			KeyNamingCase:  settings.KeyNamingCase,
 			ArgsOnSepLines: settings.ArgsOnSepLines,
 		}
+		if settings.ContextOnly && settings.Context == "" {
+			opts.ContextOnly = "all"
```

Comment:
Should we print a deprecation notice here?

Reply:
the deprecated and the fallback you should be handled inside the `Loader`
https://github.com/golangci/golangci-lint/blob/44c070af0829b1f87ecf754e2cccc4e74a22b27f/pkg/config/loader.go#L358